src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Windows.txt

```diff
@@ -0,0 +1,74 @@
+﻿@ECHO OFF
+SETLOCAL
+
+SET PACKAGE_DIR=%1
+SET EXECUTION_DIR=%2
+
+IF DEFINED PACKAGE_DIR ( echo Using %PACKAGE_DIR% as folder for resolving package dependencies.) ELSE (
+echo Please specify an package source directory using PACKAGEROOT parameter
+goto ShowUsage
+)
+
+if "%EXECUTION_DIR%" == "" (set EXECUTION_DIR=%~dp0)
+echo Executing in %EXECUTION_DIR% 
+
+:: ========================= BEGIN Test Execution ============================= 
+:RunTests
+
+:: Copy over dependencies... 
+GOTO CopyFiles
+:FileCopySucceeded
+
+set CORE_ROOT=core_root
+
+echo Running tests... Start time: %TIME%
+echo Command: [[TestRunCommand]]
+pushd %EXECUTION_DIR%
+call [[TestRunCommand]]
+popd
+echo Finished running tests.  End time=%TIME%, Exit code = %ERRORLEVEL%
+EXIT /B %ERRORLEVEL%
+:: ========================= END Test Execution =================================
+
+:: ========================= BEGIN Copying files  =============================== 
+:CopyFiles
+
+echo Hard linking dependent files... 
+:: Format here is: call :copyandcheck Path1 Path2 || GOTO EOF
+[[CopyFilesCommands]]
+
+IF /I "%CD%" NEQ "%EXECUTION_DIR%" (
+xcopy /e /q /y %CD%\*.* %EXECUTION_DIR%
+)
+
+echo Finished linking needed files, moving to running tests.
+GOTO FileCopySucceeded
+:: ======== Link a file into specified directory, checking for its existence ====
+:copyandcheck
+:: Until the PrereleaseResolveNuGetPackageAssets task is changed, publishtests.targets will 
+:: have already copied the files.  So for local execution, ignore this for now.
+IF EXIST %2 (
+exit /b 0
```

Comment:
I assume this exit is just exiting from the function call correct? It seems like below you are making the assumption that exit is existing the script all together. Without some testing I'm unclear what happens here.